### PR TITLE
Update table

### PR DESCRIPTION
**Pull request by Zuhaib042**
### Vet Clinic milestone-02
**Summary**
- I inserted more data and add a species column to the table of animals.
- Then I wrote complex queries to fetch data from the table. 
- In some queries, I used SQL transactions to place a guard clause in case of a mishap while handling data in the table.

**Here are the screenshots of the database in real-time**

vet-clinic3

vet-clinic4

vet-clinic5

vet-clinic6

### CODE REVIEW

**Comment by sja-thedude** [comment]
Hi @Zuhaib042,

I tried contacting you on slack but you were not available. If you can update your readme with the new template given [here](https://github.com/microverseinc/readme-template#readme) in the next 5 mins then I will proceed else invalidate it.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @Zuhaib042, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have an updated readme. Kindly make sure you have the updated readme.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: data.sql [hunk 1]
```diff
@@ -9,3 +9,10 @@ VALUES ('Pikachu', date '2021-01-07', 1 , 'false', 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Devimon', date '2017-05-12', 5 , 'true', 11);
 
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', date '2020-02-08', 0, 'false', 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', date '2021-11-15', 2, 'true', 5.7);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', date '1993-04-02', 3, 'false', 12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', date '2005-06-12', 1, 'true', 45);
```
Comment (Davidosky007) [inline data.sql]: * Kindly fix these weights, according to the weight provided in the project requirement you have them in negative numbers 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Zuhaib042 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* Well written PR description 👍 
* Good looking readme file 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
* You are required to Add a column species of type string to your animals table. Modify your schema.sql file. Kindly fix this as currently your animal table is missing the species column 
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Zuhaib042** [comment]
vet-clinic-update

**Comment by arslanbisharat** [comment]
Hi @Zuhaib042,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**